app/src/main/java/com/worldsnas/starplayer/di/MusicListComponent.kt

```diff
@@ -14,5 +18,7 @@ interface MusicListComponent {
         fun build(): MusicListComponent
 
         fun appComponent(appComponent: AppComponent): Builder
+        @BindsInstance
+        fun contextProvider(context: Context): Builder
```

Comment:
in context leak mishe.
benazaram behtare context Application ro az `AppComponent` begirid va be contentResolver bedid